src/pages/core/guides/immutability.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Immutability in MPL Core
+metaTitle: Core - Immutability
+description: This Guide describes the different immutability layers of MPL Core
+---
+
+## What is Immutability?
+In the general context of Digital Assets immutability is often used referring to the metadata of a Token or NFT. In the past this was asked for by communities to make sure that a bought Asset can not be changed in the future. With the additional functionality MPL Core offers it can make sense to add additional immutability features. This guide aims to provide information about those different options and how they can be used to tailor the immutability of a digital Asset to the needs of a project.
```

Comment:
Nit: I think "purchased Asset" is slightly more grammatically correct than "bought Asset"